### PR TITLE
Added missing IsWBFluidEnabled Parameter

### PR DESCRIPTION
**Pull request by FaganSC**
Added missing IsWBFluidEnabled Parameter from documentation to enable or disable Whiteboard per the Microsoft documentation
https://support.microsoft.com/en-us/office/enable-microsoft-whiteboard-on-onedrive-for-business-2fae9707-0c9b-42f8-a85f-9628b698ac82

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1193,6 +1194,26 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -IsWBFluidEnabled
+
+Sets weather Whiteboard is enabled or disabled for OneDrive for Business users. Whiteboard on OneDrive for Business is automatically enabled for applicable Microsoft 365 tenants but can be disabled.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1199]:
```suggestion
Sets whether Whiteboard is enabled or disabled for OneDrive for Business users. Whiteboard on OneDrive for Business is automatically enabled for applicable Microsoft 365 tenants but can be disabled.
```

**Review by dariomws — APPROVED** [review]
I´m ok after the @get-itips suggestion.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (FaganSC) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1199]: @get-itips thank you for that correction. Can't believe I missed that.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
Gentle reminder @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1200]: @FaganSC Can you apply the suggestion? cc @yogkumgit

**Comment by get-itips** [comment]
> Gentle reminder @get-itips

Author @FaganSC  needs to apply suggestion

**Comment by yogkumgit** [comment]
@FaganSC Gentle reminder. Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks.

**Pull request merged**